src/components/tableview/TableTable.vue

```diff
@@ -427,25 +442,62 @@ export default {
       }
       this.$set(this.pendingEdits, key, payload)
     },
+    addRowToPendingDeletes(row) {
+      row.getElement().classList.add('deleted')
+      const pkCell = row.getCells().find(c => c.getField() === this.primaryKey)
+
+      if (!pkCell) {
+        this.$noty.error("Can't delete row -- couldn't figure out primary key")       
+        return
+      }
+
+      const payload = {
+        table: this.table.name,
+        row: row,
+        schema: this.table.schema,
+        pkColumn: this.primaryKey,
+        primaryKey: pkCell.getValue()
+      }
+
+      this.$set(this.pendingDeletes, pkCell.getValue(), payload)
+    },
     async saveChanges() {
       try {
-        // throw new Error("This is an error")
-        const newData = await this.connection.updateValues(this.pendingEditList)
-        const updateIncludedPK = this.pendingEditList.find(e => e.column === e.pkColumn)
-        if (updateIncludedPK) {
+        let replaceData = false
+
+        // Commit Deletes
+        if (this.pendingDeleteList.length > 0) {
+          const deleteSuccess = await this.connection.deleteRows(this.pendingDeleteList)
+          if (deleteSuccess) {
+            replaceData = true
+          } else {
+            this.$noty.error("Error deleting rows")
```

Comment:
It should probably `return` here so as not to commit the edits?

Reply:
Sure, that makes sense. Otherwise we'd need the possibility to display multiple errors I guess.